communication/requesting-survey.md

```diff
@@ -0,0 +1,102 @@
+# Requesting a Community Survey
+
+Let us help you make your survey a success.
+
+The Kubernetes project has access to the CNCF Surveymonkey account for creating community surveys, and SIG-Contributor Experience includes people who can give advice on improving the quality of surveys, as well as promote them.  As such, what follows is the process for requesting such a community survey, in order to maximize its reach and data quality.
+
+## What's a Community Survey?
+
+Any survey requested by a Kubernetes SIG, WG, Team, or other standing Kubernetes community group (hereafter "Group") that targets some or all of the Kubernetes community and ecosystem is potentially a community survey.  As some examples, this would include:
+
+* A survey by SIG-Cluster-Lifecycle on upgrade practices, targeted at Kubernetes users
+* A survey by SIG-ContribEx on contributor events, targeted at current contributors
+* A survey by the Release Engineering Team, targeted at vendors who repackage Kubernetes
+
+Not included in community surveys are: surveys by specific vendors, surveys by other CNCF projects, surveys of target audiences outside the Kubernetes/CNCF community, or surveys that cannot be executed using SurveyMonkey and/or GoogleDocs.
+
+If the survey you want to do is a community survey, then then next thing is to submit a request.
+
+## Survey Request Process
+
+### 1. Determine Goals and Content of Survey
+
+The first step needs to happen in your community Group.  You need to determine:
+
+1. Why you want to do a survey
+2. Who you would like to answer it
+3. What specific questions you want to ask
+
+Ideally, answer those questions in that order; first decide what your goals are, next which audiences you're targeting, and finally compose the individual questions.
+
+Your survey will also need a preface which explains to the audience what the survey is for, who will get to see its raw results, and where compiled data from it will be published.
+
+### 2. Request Survey Review
+
+Put your draft survey questions in a Google Doc, HackMD page, PR against one of your Group repos, or other format that supports comments by reviewers.  Then open an [issue](https://github.com/kubernetes/community/issues) against the community repo using this template:
+
+```
+Group Requesting:
+
+Title of Survey:
+
+Goal of Survey:
+
+Target Audience:
+
+Results Recipients:
+
+Open/Close Dates:
+
+Communications Help Wanted?  Y / N
+
+Data Analysis Wanted?  Y / N
```

Comment:
We should expand on this and also scope what ContribEx can and can't do analysis-wise.

I almost err towards not offering additional data analysis past what the survey platform's tools support.

Reply:
Yeah, I was torn about this too.  Thing is, I can an do offer some additional analysis, mostly simple things like correlating several questions together.  But that's only for SIGs I participate in because I don't get requests for others.  So I wanted to offer the ability for folks to *ask*, even if we sometimes might say "no".

So ... I'll add what kind of analysis is available instead of removing this.